docs/en/serverless/projects/billing.asciidoc

```diff
@@ -11,11 +11,16 @@ Your monthly bill is based on the capabilities you use.
 When you use {obs-serverless}, your bill is calculated based on data volume, which has these components:
 
 * **Ingest** — Measured by the number of GB of log/event/info data that you send to your Observability project over the course of a month.
-* **Storage/Retention** — This is known as Search AI Lake.
-* In addition to the core ingest and retention dimensions, there is an optional charge to execute synthetic monitors on our testing infrastructure.
-Browser (journey) based tests are charged on a per-test-run basis,
-and Ping (lightweight) tests have an all-you-can-use model per location used.
+* **Retention** — Measured by the total amount of ingested data stored in your Observability project.
 
-For more information, refer to <<general-serverless-billing>>.
+Ingest and retention metering is based on the uncompressed, normalized, fully enriched data volume you ingest into your Serverless project. This allows you flexibility and consistency when choosing your preferred ingest architecture for enrichment, whether through {agent}, {ls}, OpenTelemetry, or collectors—without the need to consider possible impacts on the cost.
 
-For detailed {obs-serverless} project rates, check the https://www.elastic.co/pricing/serverless-observability[{obs-serverless} pricing page].
+NOTE: One consequence of this metering method is that the absolute volume of data reported in your usage statement can be much larger than the "raw" data size or the compressed data size "on the wire." Your monthly bill will transparently display exactly how much data has been ingested, including valuable metadata added by your data shipper, and contextual enrichment performed by your ingest processing. Don't worry, we've taken all that into account with our Serverless per GB prices, so we think you'll find Elastic Serverless to be a very cost-effective service.
+
+[discrete]
+[[synthetics-billing]]
+== Synthetics
+
+<<observability-monitor-synthetics,Synthetic monitoring>> is an optional add-on to Observability Serverless projects that allows you to periodically check the status of your services and applications. In addition to the core ingest and retention dimensions, there is a charge to execute synthetic monitors on our testing infrastructure. Browser (journey) based tests are charged per-test-run, and ping (lightweight) tests have an all-you-can-use model per location used.
+
+Refer to <<general-serverless-billing,Serverless billing dimensions>> and the https://cloud.elastic.co/cloud-pricing-table?productType=serverless&project=observability[Elastic Cloud pricing table] for more details about Elastic Observability serverless billing dimensions and rates.
```

Comment:
I'm not sure if it's totally necessary to capitalize the `serverless` in this context, but it might be good to use the attribute for consistency. 

```suggestion
Refer to <<general-serverless-billing,Serverless billing dimensions>> and the https://cloud.elastic.co/cloud-pricing-table?productType=serverless&project=observability[Elastic Cloud pricing table] for more details about {obs-serverless} billing dimensions and rates.
```